DuckDuckGo/FavoritesEmptyStateView.swift

```diff
@@ -55,5 +62,17 @@ struct FavoritesEmptyStateView<Model: FavoritesEmptyStateModel>: View {
 }
 
 #Preview {
-    return FavoritesEmptyStateView(model: FavoritesPreviewModel(), geometry: nil)
+    PreviewViewWrapper()
+}
+
+private struct PreviewViewWrapper: View {
```

Comment:
I was using Xcode16 and it gives a warning when `@State` is used inside `Preview` without @Previewable tag, but it's available only from iOS17. I figured wrapper here is the quickest solution for now.